tools/doc-generator/main.go

```diff
@@ -168,6 +175,11 @@ var (
 			structType: reflect.TypeOf(compactor.Config{}),
 			desc:       "The compactor_config configures the compactor for the experimental blocks storage.",
 		},
+		{
+			name:       "store_gateway_config",
+			structType: reflect.TypeOf(storegateway.Config{}),
+			desc:       "The store_gateway_config configures the store-gateway service used by the experimental blocks storage.",
+		},
```

Comment:
Exposing this may be little premature, given that this component is not yet finished. /cc @pracucci

Reply:
You're actually right. Let's hide it 👍

Reply:
There is no way to hide this by doc-generator.
The only way to do that is add `doc:"hidden"` tag on Config struct.

You can check current config file docs looks not so good.
https://cortexmetrics.io/docs/configuration/configuration-file/#supported-contents-and-default-values-of-the-config-file

Reply:
Yeah, definitely. We did mean to use `doc:"hidden"` on the store gateway config.

Reply:
I've added doc:hidden